platform/local/dnsmasq.go

```diff
@@ -147,6 +225,120 @@ func newSegment(s byte) (*Segment, error) {
 		return nil, fmt.Errorf("LinkSetUp() failed: %v", err)
 	}
 
+	// we first create an unique virtual ethernet pair in current network namespace accross the multiple kola instances
+	// we use linux network random port attribution to assert the uniqueness of the IP range in order to avoid IP
+	// range clashes
+	listener, err := net.Listen("tcp", ":0")
```

Comment:
This needs to be done in the host network namespace (`rootExit, err := ns.Enter(root)` and leave again)